### PR TITLE
Describe how to use WeMo long press event triggers

### PR DESCRIPTION
**Pull request by esev**
## Proposed change

Describe how to use WeMo long press event triggers

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45503

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wemo.markdown [hunk 1]
```diff
@@ -114,3 +114,30 @@ There are several services which can be used for automations and control of the
 | `turn_on` | Calling this service will turn the humidifier on and set the speed to the last used speed (defaults to medium, entity_id is required).
 | `wemo.set_humidity` | Calling this service will set the desired relative humidity setting on the device (entity_id is a required list of 1 or more entities to set humidity on, and target_humidity is a required float value between 0 and 100 (this value will be rounded down and mapped to one of the valid desired humidity settings of 45, 50, 55, 60, or 100 that are supported by the WeMo humidifier)).
 | `wemo.reset_filter_life` | Calling this service will reset the humdifier's filter life back to 100% (entity_id is a required list of 1 or more entities to reset the filter life on). Call this service when you change the filter on your humidifier.
+
+## Long Press Events and Triggers
+
+For WeMo Light Switches and Dimmers, pressing the button on the device for two seconds will activate a long press event. The long press can can trigger an automation
```
Comment (frenck) [inline source/_integrations/wemo.markdown]:
```suggestion
For WeMo Light Switches and Dimmers, pressing the button on the device for two seconds will activate a long press event. The long-press can trigger an automation
```

File: source/_integrations/wemo.markdown [hunk 2]
```diff
@@ -114,3 +114,30 @@ There are several services which can be used for automations and control of the
...
+  alias: Toggle amplifier power
+  trigger:
+  - platform: event
+    event_type: wemo_subscription_event
+    event_data:
+      type: LongPress
+      entity_id: Living Room
+  action:
+  - service: media_player.toggle
+    entity_id: media_player.amplifier
```
Comment (frenck) [inline source/_integrations/wemo.markdown]:
```suggestion
- id: long_press_living_room
  alias: "Toggle amplifier power"
  trigger:
    - platform: event
      event_type: wemo_subscription_event
      event_data:
        type: LongPress
        entity_id: Living Room
  action:
    - service: media_player.toggle
      target:
        entity_id: media_player.amplifier
```

File: source/_integrations/wemo.markdown [hunk 3]
```diff
@@ -114,3 +114,30 @@ There are several services which can be used for automations and control of the
...
+```yaml
+# Example automation
+- id: long_press_living_room
+  alias: Toggle amplifier power
+  trigger:
+  - platform: event
+    event_type: wemo_subscription_event
+    event_data:
+      type: LongPress
+      entity_id: Living Room
```
Comment (frenck) [inline source/_integrations/wemo.markdown]: This seems an invalid `entity_id`?

File: source/_integrations/wemo.markdown (hunk 1 above)
Comment (MartinHjelmare) [inline source/_integrations/wemo.markdown]:
```suggestion
For WeMo Light Switches and Dimmers, pressing the button on the device for two seconds will activate a long press event. The long press can trigger an automation
```

Comment (esev) [reply source/_integrations/wemo.markdown]: Fixed

File: source/_integrations/wemo.markdown (hunk 3 above)
Comment (esev) [reply source/_integrations/wemo.markdown]: Thanks for catching that. Replaced `entity_id` with `name`.

**Review by frenck — APPROVED** [review]
Thanks, @esev 👍

**Pull request merged**